docs/standard/data/xml/xdr-validation-with-xmlschemacollection.md

```diff
@@ -18,7 +18,7 @@ If the XML-Data Reduced (XDR) schema you are validating against is stored in the
 
 For example, if the root element of the XML document is `<bookstore xmlns="urn:newbooks-schema">`, when the schema is added to the **XmlSchemaCollection** it references the same namespace, as follows:
 
-```
+```csharp
```

Comment:
we probably need to add a second block, one for csharp with the ; at the end

```suggestion
```vb
```